.kuzzlerc

```diff
@@ -116,14 +165,60 @@
           }
         },
         "%kuzzle": {
```

Comment:
hmm for now, _canCreate and _canDelete policies are true by default, so we have to set them explicitly to "false" if we want to deny them.
This has to be fixed in a next stage, but it's not about the current PR ;)